zenodo/config.py

```diff
@@ -1071,13 +1087,23 @@ def _(x):
             provisional_communities=terms_filter('provisional_communities'),
             bounds=geo_bounding_box_filter(
                 'bounds', 'locations.point', type='indexed'),
+            publication_date=range_filter(
+                    field='publication_date',
+                    format='yyyy-MM-dd',
```

Comment:
```suggestion
                    format='yyyy',
```

To match the above format, and to keep in line with the rounding (i.e. otherwise it doesn't matter if one passes `2020-05-12` or `2020-07-20`, since it will be rounded to the year)